### PR TITLE
Fix A11y issues with Pipelines page, runs page and Create/Duplicate Run

### PR DESCRIPTION
**Pull request by manaswinidas**
Fixes [RHOAIENG-1750](https://issues.redhat.com/browse/RHOAIENG-1750)

## Description

Fixes all a11y issues flagged by axe Devtools in Pipelines Table page, Runs Table rows, Create/Duplicate page

Before: 
Screenshot 2024-01-25 at 8 43 33 PM

After: 

Screenshot 2024-01-25 at 8 45 24 PM

Screenshot 2024-01-25 at 8 46 22 PM

## How Has This Been Tested?

1. Go to Pipelines page, Runs page, Create Run pages and try running an axe Devtools SCAN 
2. Pull this PR and run an axe Devtools SCAN

## Test Impact

Tests being covered in other Pipeline logs/versions tests PR

## Request review criteria:

Self checklist (all need to be checked):
- [X] The developer has manually tested the changes and verified that the changes work
- [X] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [X] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [X] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Review by DaoDaoNoCode — APPROVED** [review]
Tested and lgtm.
/lgtm

**Comment by DaoDaoNoCode** [comment]
@manaswinidas Can you resolve conflicts? It should be OK just have `Checkbox` as the `aria-label`

**Comment by manaswinidas** [comment]
WIP to fix this issue in the Pipeline Run Bottom Drawer page: 
Screenshot 2024-01-30 at 6 02 12 PM

File: frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDrawerBottomTabs.tsx [hunk 1]
```diff
@@ -9,8 +9,8 @@ import PipelineRunTabParameters from './PipelineRunTabParameters';
 
 export enum RunDetailsTabs {
   DETAILS = 'Details',
-  PARAMETERS = 'Input parameters',
-  YAML = 'Run output',
+  PARAMETERS = 'Inputparameters',
+  YAML = 'Runoutput',
 }
```
Comment (DaoDaoNoCode) [inline frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDrawerBottomTabs.tsx:15]: I prefer to add a enum here like
```
export enum RunDetailsTabs {
  DETAILS = 'details',
  PARAMETERS = 'input-parameters',
  YAML = 'run-output',
}

const RunDetailsTabTitles = {
  [RunDetailsTabs.DETAILS] :'Details',
  [RunDetailsTabs.PARAMETERS]: 'Input parameters',
  [RunDetailsTabs.YAML]: 'Run output',
}
```
and you only need to change title to `title={RunDetailsTabTitles[tab]}` below, which is more clear.

**Comment by DaoDaoNoCode** [comment]
You may need to update the tests in `frontend/src/__tests__/cypress/cypress/pages/pipelines/topology.ts` from line 18-25 findByTestId to make it correctly find the test id you changed because the tab is changed. I think the best way is to change those to `bottom-drawer-tab-${RunDetailsTabs.xxx}`

**Comment by manaswinidas** [comment]
Changed the test-id to `RunDetailsTabsTitles[tab]`

**Review by DaoDaoNoCode — APPROVED** [review]
/lgtm

**Pull request merged**